code_review/dp/code_review_daw_parms.Rmd

```diff
@@ -0,0 +1,280 @@
+---
+title: "code_review_daw_parms"
+author: "Daniel Petrie"
+date: "2024-08-16"
+output: html_document
+editor_options: 
+  chunk_output_type: console
+---
+
+Packages/WD/DATA
+
+```{r}
+library("ggplot2") #For plotting
+library("GGally") #ggpairs()
+library("tidyverse") #Wranglin
+library("dplyr") #Wranglin
+library("lme4") #MLM
+library("lmerTest") #p-vals
+library("ggeffects") #For marginal/conditional effects plots
+#library("margins") #For margins() function. Useful for comparing margins from ggeffects
+library("marginaleffects") #For hypothesis_test()
+library("mgcv") #GAMM
+library("LNCDR") #waterfall plot, lunaize plots 
+library("gratia") #mgcv companion package. Using draw among other funcs.
+library("psych") #Descriptives
+
+#Working directory (change to something better)
+setwd("C:/Users/djpet/Documents/tutorials/code_review")
+
+#Beh DAW files /Volumes/Hera/Dan/daw/analysis/txt
+luna_7t <- read.csv("luna_7tbrainmech.csv", header = TRUE)
```

Comment:
[vroom::vroom](https://vroom.r-lib.org/) is useful for reading in large CSVs

Reply:
any idea if `readr::read_csv` is similiar?